lib/fathead/pandas/fetch.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+VERSION=0.20.1
```

Comment:
@souravsingh Strangely, 0.20.1 version now throws 404 error. 0.20.3 is the latest stable build and documentation for it is available. We should fix this.

Alternatively, we can point to stable documentation zip file and remove version altogether. Link for stable documentation would be `http://pandas.pydata.org/pandas-docs/stable/pandas.zip`